### DISCUSSION TITLE
短数字无法检测问题

### DISCUSSION THREAD

**Discussion by yhl41001** [PaddlePaddle/PaddleOCR#13108] [Q&A] [answered]
#### 问题描述 / Problem Description
我尝试用我们的OCR来识别电梯按钮，但是发现对1到9这类单个数字的检测召回率非常低，有办法优化该问题吗？

#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:

**Comment by jingsongliujing** [comment]
假如就单个数字的话，可以用目标检测paddledetection套件直接标注检测：  https://github.com/PaddlePaddle/PaddleDetection

**Reply by yhl41001** [reply]
目前模型已经具备了这个能力还是需要重新标注数据训练？

**Comment by UserWangZz** [comment, marked as answer]
有相关badcase吗？或者尝试使用小数据集微调一下检测模型？

**Reply by yhl41001** [reply]
明白了，就是需要自己来微调一下比较好。我现在还没数据集，等我看一下自己能不能解决吧，多谢回复！
